package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "combo =====",
   "engines": {
-    "node": "6.2.0"
+    "node": "6.x"
```

Comment:
It's okay if we bump the node version, but let's use strict versioning to ensure repeatability.